tests/backup/backup_helper.go

```diff
@@ -2672,6 +2672,13 @@ func upgradeStorkVersion(storkImageToUpgrade string) error {
 			return err
 		}
 		storageSpec.Spec.Stork.Image = finalImageToUpgrade
+
+		// Check to reset customImageRegistry to blank as in case of ibm it'll be icr.io/ext/ and not
+		// docker.io/ which causes issues when we try to install stork which is not pushed to icr.io/ext
+		if os.Getenv("CLUSTER_PROVIDER") == "ibm" {
+			storageSpec.Spec.CustomImageRegistry = ""
```

Comment:
How about we log the value of storageSpec.Spec.CustomImageRegistry before making it empty?

Reply:
How will that be helpful?

Reply:
I am not sure, but for debugging purposes, logging the original value of storageSpec.Spec.CustomImageRegistry before overriding it could be helpful to verify it was not already empty.

Reply:
When you deploy from IBM catalog it will always have the value and hence it won't be empty. Hence this case won't occur.